requirements.txt

```diff
@@ -1,7 +1,6 @@
 # These are required to run the tests.
 nose
 mock
-fabric
-Django<1.7
```

Comment:
Django will be added by tox, to prevent duplicate requirement it was removed from txt file.